src/Microsoft.Internal.IO/src/Configurations.props

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      net461-Windows_NT;
```

Comment:
You shouldn't include redundant builds in the package.  Versionless configs are needed for build-from-source, but only included versioned configs in the package if they are the same config as versioned.

Reply:
Redundant how? I was trying to target 461 and 472.

Reply:
It's redundant because your DLL will have no differences between these two builds.

Reply:
Yeah, but no-one cares on a DLL.  VS may check it for loose binary refs, but they won't complain if you're using an older version than what you're targeting.  On an exe on desktop it'll drive quirking, but thats it.

As a general rule we try to avoid insignificant cross-compiles in the package.